loops_cc.py

```diff
@@ -0,0 +1,74 @@
+#!/usr/bin/env python3
+# -*- coding: utf-8 -*-
+"""
+Created on Thu May 10 15:34:22 2018
+
+@author: amyhaddad
+"""
+
+#Use a for loop to print the numbers 1 to 20, inclusive
+for value in range(1,20):
+    print(value)
+
+
+#Make a list of numbers from 1 to one million and then use a for loop to print the numbers
+numbers = list(range(1, 1000000))
+print(numbers)
+
+numbers = []
+for value in range(1, 1000000):
+    numbers.append(value)
```

Comment:
I'd add a line break after line 20 to separate the for loop from the next statement (since they are separate).